.travis.yml

```diff
@@ -38,6 +38,8 @@ before_cache:
 
 matrix:
   include:
+    - compiler: "ghc-8.8.1"
+      addons: {apt: {packages: [ghc-ppa-tools,cabal-install-2.4,ghc-8.8.1], sources: [hvr-ghc]}}
```

Comment:
Use `haskell-ci` to regenerate this file instead of editing it manually (see [here](https://github.com/haskell/unix/pull/129)).

Reply:
Should be done now.